packages/forms/docs/20-code-editor.md

```diff
@@ -14,3 +14,17 @@ CodeEditor::make('code')
 ```
 
 <AutoScreenshot name="forms/fields/code-editor/simple" alt="Code editor" version="4.x" />
```

Comment:
Now that there isn't syntax highlighting out of the box, the code used to generate this screenshot in `FieldsDemo.php` needs to be updated to ensure it has the correct language